dbstore.go

```diff
@@ -409,30 +470,41 @@ func (p *persister) update(ws *Worksheet) error {
 
```

Comment:
I played around with this with our existing endpoints and handlers for the escrows hurdle and couldn't get any values to show up in the `worksheet_parents` table until I commented out the block starting with `// no change, i.e. only the version would change`.

Reply:
I'm curious why the children update the worksheet_parents table, instead of letting the parent handle that?

Reply:
Thanks for reproducing the bug we were talking about! Incorporated your code in a new test case https://github.com/helloeave/worksheets/pull/79/commits/5e93c08f5b027ad0a24e9a1585a1ecdd20c4593e.

Next, work on fixing this!

Reply:
@alexandrinaw Fixed! You are right, it's much more correct and clear to have parents own the `worksheet_parents` relationship. Since it is pointers flowing from the parents, capturing changes there, and updating makes things much more precise and easy to track. It also ensure we're concurrency safe since there is a clear owner (the parent) and that any update to the collection is synchronized on the parent's `worksheets.version` column.